load_testing/locust/.gitignore

```diff
@@ -0,0 +1 @@
+*/loadTestConstants.py
```

Comment:
Can you update this to ignore .env as well?

Reply:
.env added